pkgs/top-level/python-packages.nix

```
@@ -187,6 +187,8 @@ in {
 
   argon2_cffi = callPackage ../development/python-modules/argon2_cffi { };
 
+  aria2p = callPackage ../development/python-modules/aria2p { inherit (pkgs) aria2; };
```

you'll want the poetry from the top-level package set
```suggestion
  aria2p = callPackage ../development/python-modules/aria2p { inherit (pkgs) aria2 poetry; };
```

---

Done, thanks.